docs/GetStarted/getting_started_linux_bazel.md

```diff
@@ -31,13 +32,19 @@ We recommend Clang. GCC is not fully supported.
 $ sudo apt install clang
 ```
 
-Set environment variables:
+Set environment variables for Bazel:
```

Comment:
Is this still needed at the moment? I think Stella introduced a [script](https://github.com/google/iree/blob/master/configure_bazel.py) that handles this?

Reply:
It looks like they aren't. I'll remove them from the linux bazel getting started too then.

Reply:
Okay, I don't know what the problem is, but `./build_tools/bazel/build_tensorflow.sh` doesn't work without these environment variables, so I'm going to add them back for now.